### DISCUSSION TITLE
Multiple SX1262 modules

### DISCUSSION THREAD

**Discussion by mfvmx** [jgromes/RadioLib#377] [Q&A]
I have a board that I wanted to setup to receive three Lora frequencies, which it does fine, but the moment I send a message out it freezes and all channels stop receiving, any idea what could cause this? I will be adding code shortly.

**Comment by aaseagrass** [comment]
I had set up before a multinode LoRa where in there is Master Node (server) and Slave nodes. I noticed that if 2 slave nodes send data at the same time, the Master node will only be able to receive either data from the 2 slaves but not both.

Better to introduce delay or set time in every node you have when it will send data to make sure that Master will receive all of it.
